BUILD.gn

```diff
@@ -28,6 +28,9 @@ config("vulkan_headers_config") {
   if (is_fuchsia) {
     defines = [ "VK_USE_PLATFORM_FUCHSIA" ]
   }
+  if (is_mac) {
+    defines = [ "VK_USE_PLATFORM_MACOS_MVK" ]
```

Comment:
Does this have to be the MVK define? I thought Vulkan implementations on macOS were headed toward VK_EXT_metal_surface/VK_USE_PLATFORM_METAL_EXT

Reply:
Swiftshader implements the MVK one, but I just tested a patch to implement the EXT one and it works. I'll use the EXT one in this patch